src/store/SettingsStore.js

```diff
@@ -23,6 +23,12 @@ const COLOR_PALETTE = [
   "#4e79a7", // blue
 ];
 
+function randomChromaColor(){
+  const randomSaturation = Math.random() * 0.2 + 0.6; 
```

Comment:
@hamzakammar Could you add a comment at the top of this line stating the ranges of saturation and lightness that this translates to? It can help with future maintenance.